test-clients/src/main/java/com/hedera/services/bdd/suites/perf/CryptoTransferLoadTest.java

```diff
@@ -77,6 +79,9 @@ private HapiApiSpec runCryptoTransfers() {
 						withOpContext((spec, ignore) -> settings.setFrom(spec.setup().ciPropertiesMap())),
 						logIt(ignore -> settings.toString())
 				).when(
+						fileUpdate(APP_PROPERTIES)
+								.payingWith(GENESIS)
+								.overridingProps(Map.of("hapi.throttling.buckets.fastOpBucket.capacity", "10000")),
```

Comment:
The total throttling value is sum of slowOpBucket and fastOpBucket. With this change, the total will be 16000. I guess this will be ok.

Reply:
@ljianghedera Thank you! Will modify it to 4000 as sum will be 10K.

Reply:
That's good. Thx.